Makefile

```diff
@@ -18,7 +18,9 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.14
+# release images are available at:
+# console.google.com/gcr/images/cloud-foundation-cicd/global/cft/developer-tools?project=cloud-foundation-cicd
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := :0.15.9
```

Comment:
This versioning is based on the images in: https://pantheon.corp.google.com/gcr/images/cloud-foundation-cicd/global/cft/developer-tools?project=cloud-foundation-cicd

It used to be that this version matched the terraform version installed inside the docker iamge. But recently it seems to have diverged a bit